### DISCUSSION TITLE
Inline field creation/editing in the field layout designer

### DISCUSSION THREAD

**Discussion by nexflo** [craftcms/cms#8530] [Ideas] [answered]
While in the field manager, it’s cumbersome actually adding fields. An add field button just like https://straightupcraft.com/craft-plugins/quick-field quick-field provided should be a core feature.

**Comment by craftedsystems** [comment]
The plugin can be found here:
https://github.com/spicywebau/craft-quick-field

**Comment by brandonkelly** [comment, marked as answer]
This has officially made it in for Craft 5 🎉  (#14260)

5.0.0-alpha.11 was just tagged with it.
